zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -73,10 +70,11 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
       createdTask.outputDir.set(linkOutputDir.parentFile.resolve("${linkOutputDir.name}Zipline"))
     }
 
-    project.tasks.withType(KotlinWebpack::class.java).configureEach { kotlinWebpack ->
-      if (kotlinBinary.mode.toString().equals(kotlinWebpack.mode.toString(), ignoreCase = true)) {
-        kotlinWebpack.dependsOn(compileZiplineTaskName)
-      }
+    val serveTaskName = "serve${kotlinBinary.mode.toString().lowercase().capitalized()}Zipline"
+    project.tasks.register(serveTaskName, ZiplineServeTask::class.java) { createdTask ->
+      createdTask.description = "Serves Zipline files"
+      createdTask.dependsOn(compileZiplineTaskName)
+      createdTask.inputDir.set(ziplineCompileTask.get().outputDir)
```

Comment:
You should never need to write `dependsOn` with property-based tasks.
```suggestion
      createdTask.inputDir.set(ziplineCompileTask.map { it.outputDir })
```
(This diff is just a guess, but it should be something like this)

Reply:
Amazing - I was wondering why I was getting two builds every time I changed a file. Ended up coming up with a hack to pass the serve task a string instead of a Directory, but this is much better. Speed gains are even more now!